extract_i18n.sh

```diff
@@ -8,7 +8,7 @@ declare -a arr=(
     "eng_GB" "tha_TH" "ind_ID" "swa_TZ" "hin_IN" "fre_FR" "spa_CO" "spa_MX"
     "spa_AR" "mar_IN" "tel_IN" "por_PT" "mal_IN" "guj_IN" "tam_IN" "urd_IN"
     "urd_PK" "ben_IN" "ben_BD" "nep_NP" "fil_PH" "kan_IN" "ori_IN" "pan_IN"
-    "asm_IN")
+    "asm_IN", "km_KH" )
```

Comment:
@bruskiza we're using the ISO 639-2 because not all languages are covered by the 2-code spec. http://www.loc.gov/standards/iso639-2/php/code_list.php